.bc-exclude.php

```diff
@@ -59,5 +58,13 @@
         // Fix for promotion discount entity property initialization error - necessary to prevent runtime errors
         preg_quote('CHANGED: Type of property Shopware\Core\Checkout\Promotion\Aggregate\PromotionDiscount\PromotionDiscountEntity#$usageKey changed from string to string|null', '/'),
         preg_quote('CHANGED: The parameter $usageKey of Shopware\Core\Checkout\Promotion\Aggregate\PromotionDiscount\PromotionDiscountEntity#setUsageKey() changed from string to string|null', '/'),
+
+        // Symfony Console Command constructor changes - internal Symfony component changes
+        preg_quote('REMOVED: Method Symfony\Component\Console\Command\Command#__construct() was removed', '/'),
```

Comment:
you can also simplify that by using 

```suggestion
        preg_quote('Symfony\Component\Console\Command\Command#__construct()', '/'),
```
should also work. no need to be that specific :grin:

Reply:
```
[BC] ADDED: Parameter filesystem was added to Method __construct() of class Symfony\Component\Console\Command\Command
```
It would leave this one then.

`preg_quote('Symfony\Component\Console\Command\Command', '/'),` would do

Reply:
I am not sure that having this generic ignore is nice - are we sure it will not skip some important bc break?

Reply:
So @h1k3r and @mitelg what's the agreement on this?

Reply:
I don't see how it would skip important bc on objects extending Symfony Command and, neither impact on Symfony itself since it's vendor 🤔

Reply:
`preg_quote('Symfony\Component\Console\Command\Command', '/'),` is definetely to generic :see_no_evil: 
then we need two lines I guess :slightly_smiling_face:

Reply:
Alright! 👍

Reply:
I like initial version or version proposed by Telgi (because we do not expect breaks for constructor). _If I correctly understand_ how ignores work - they search for substring match. So any message referencing symfony Command class will be ignored.

Reply:
Well we already don't hesitate to discard anything related to DBAL with `'Doctrine\\\\DBAL',` on line 17, but anyway I will set it back to a more narrow pattern